classes/teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age:, specialization:, name: 'Unknow', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] Sure to arrange your `super` method arguments as pointed out earlier.

- [ ] Kindly use the right notation

Reply:
Done